api/backend_test.go

```diff
@@ -1637,7 +1637,6 @@ func TestRestoreAccountAndLogin(t *testing.T) {
 		Mnemonic:    "test test test test test test test test test test test test",
 		FetchBackup: false,
 		CreateAccount: requests.CreateAccount{
-			DisplayName:        "Account1",
```

Comment:
Would be nice to keep a test with `DisplayName` as well.
Such case is possible when `FetchBackup` is false, meaning that the user is importing a new seed phrase, which was never a Status account. Then the user is setting the `DisplayName` right away.

Reply:
Thank you, created a separate test for testing the fix `TestRestoreAccountAndLoginWithoutDisplayName`

Reply:
Thank you!